Source/ULibs.TinyJsonDeser/JsonDeserializer.cs

```diff
@@ -1,6 +1,7 @@
 ﻿#nullable enable
 using System;
 using System.Collections.Generic;
+/***using System.Diagnostics.CodeAnalysis;***/

```

Comment:
What does this do out of interest?

Reply:
When the NuGet package is made, any comments surrounded by `/***` and `***/` get uncommented. Thus this particular line imports the `System.Diagnostics.CodeAnalysis` namespace, and a bit further down the `[ExcludeFromCodeCoverage]` class attribute is also uncommented. This ensures that the test code coverage of a project doesn't drop when this package is referenced, because the consuming project is unlikley to have tests for this source code.

Reply:
See https://github.com/red-gate/MicroLibraries/blob/master/.build/build.ps1#L162

Reply:
Oh, maybe I wasn't clear. We want the `[ExcludeFromCodeCoverage]` attribute to appear in the source when we ship it as a NuGet source package, but we don't want it when actually developing each library, so that we can still actually measure the code coverage of its test suite.

Reply:
Yes that makes sense - thanks!